<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FajoutProd</class>
 <widget class="QDialog" name="FajoutProd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>736</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>651</width>
    <height>736</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ajout Produit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Ajouter un Produit</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nom du produit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomprod_text">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_cat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Categorie</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_categories">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Electricité</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox_vendre">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>A vendre</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_acheter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>A acheter</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tab_ajout_prod">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>601</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="information">
     <attribute name="title">
      <string>Information</string>
     </attribute>
     <widget class="QLineEdit" name="cout_text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Coût de Production</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_produit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="barcode_text">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EAN13 Barcode</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Identifiant Projet</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ident_proj">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date Debut Production</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date Fin Production</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_debut">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_fin">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_duree">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>150</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Durée de Production</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="inventaire">
     <attribute name="title">
      <string>Inventaire</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Stock</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quantité disponible</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Entrant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sortant</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>20</width>
        <height>81</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="qte_dispo">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="entrant_text">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sortant_text">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Statut</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Statut</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_projectstate">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Choisir...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>En Developpement</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fin de Cycle</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Responsable</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_resp">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Site</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>190</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Entrepot</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>160</y>
        <width>20</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Site de Stockage</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_entrepot_stock">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>190</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Choisir...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Paris</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nice</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Strasbourg</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="combo_site_stock">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>160</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Choisir...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nuclear Site</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Thermal Site</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="quantite">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quantité</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="but_quitter">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_creer_prod">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Creer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_afficher_prod">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_refresh">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Raffraichir</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_affich">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>631</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Affichage des Produits</string>
   </property>
   <widget class="QTableView" name="table_produits">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>611</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections/>
</ui>
